demystifying_command_line/demystifying_command_line.md

```diff
@@ -0,0 +1,251 @@
+
+* You have to be a computer genius to use the command line, 
+* The consequences of getting a command wrong are severe, and 
+* People who use the command line have all the things they need to type memorized (see also: they are geniuses).
+
+This means that many people who are new to the command line interface might find the idea of working in the CLI intimidating.  We hope we can explain the pros and cons of the command line so you can make a decision based on what's most useful to you, instead of on misconceptions about how difficult or dangerous working in the command line is.
+
+![A three-panel cartoon. A computer user, with username robm, types the command `sudo su` into the computer.  The computer responds with the warning "robm is not in the sudoers file.  This incident will be reported."  The  user asks another person, "Hey - who does sudo report these 'incidents' to?"  The other person replies, "You know, I've never checked." In the last panel, Santa Claus is seen adding a name to his "naughty" list.](https://imgs.xkcd.com/comics/incident.png"[Incident](https://xkcd.com/838/) comic by xkcd, [CC BY-NC 2.5](https://xkcd.com/license.html).")
+
+
+## What Is a CLI (Command Line Interfaceb?)
+
+Let's start by going over what the command line interface is.
+
+Before slick graphical interfaces, before there was such a thing as a computer mouse, the usual way to interact with a computer was by typing commands through a keyboard.  The command line was usually at the bottom of the screen, with a prompt (perhaps `C:>`) and a blinking cursor that was waiting for the user to type a command.  The user could type a command, in a special kind of code that the computer could understand, and the computer would do that thing.  
+
+For example, in a UNIX or Linux computer (and later in Macs), the user might type `ls` to list the files in a particular location.  In MS-DOS (and later in Windows), a user might type `dir` to do the same thing.
+
+Here's a screenshot of a very common thing you might want to do: list files in a particular directory.  In the image, the user is working in a directory called `bias_variance_tradeoff` and asks the computer to list the files in that directory using the command `ls`.  Three things are listed: two files, and a directory called `media`.  The user then asks for a list of the files in the `media` directory by typing `ls media`, and six file names are shown.
+
+![A terminal window in which the user is working in a directory called `bias_variance_tradeoff`.  The command `ls` is followed by a file listing of two files, and a directory called `media`.  The next command is `ls media`, and six file names are shown.](media/ls.png)
```

Comment:
```suggestion
```
Consider removing this to simplify the content a bit, and to avoid opening various jargon-y cans of worms (what is UNIX? Or Linux? What's MS-DOS?) 

I don't think we need to show any actual commands in this content, since this is supposed to be a short and sweet overview, and our command line 101 does a pretty good job of introing folks to this.